api/alembic/versions/fa4b3ecb57fe_populate_tpi_fuel_area_table.py

```diff
@@ -0,0 +1,67 @@
+"""Populate tpi_fuel_area table
+
+Revision ID: fa4b3ecb57fe
+Revises: 1e6932096921
+Create Date: 2025-01-21 10:04:48.838953
+
+"""
+from alembic import op
+import geoalchemy2
+import sqlalchemy as sa
+
+from sqlalchemy.orm.session import Session
+from app.auto_spatial_advisory.process_tpi_fuel_area import calculate_masked_tpi_areas
+
+
+# revision identifiers, used by Alembic.
+revision = "fa4b3ecb57fe"
+down_revision = "1e6932096921"
+branch_labels = None
+depends_on = None
+
+shape_type_table = sa.Table("advisory_shape_types", sa.MetaData(), sa.Column("id", sa.Integer), sa.Column("name", sa.String))
+
+shape_table = sa.Table(
+    "advisory_shapes",
+    sa.MetaData(),
+    sa.Column("id", sa.Integer),
+    sa.Column("source_identifier", sa.String),
+    sa.Column("shape_type", sa.Integer),
+    sa.Column("geom", geoalchemy2.Geometry),
+)
+
+tpi_area_table = sa.Table(
+    "tpi_fuel_area",
+    sa.MetaData(),
+    sa.Column("advisory_shape_id", sa.Integer),
+    sa.Column("tpi_class", sa.Enum("valley_bottom", "mid_slope", "upper_slope", name="tpiclassenum")),
+    sa.Column("fuel_area", sa.Float),
+)
+
+
+def get_fire_zone_unit_shape_type_id(session: Session):
+    statement = shape_type_table.select().where(shape_type_table.c.name == "fire_zone_unit")
+    result = session.execute(statement).fetchone()
+    return result.id
+
+
+def get_fire_zone_units(session: Session, fire_zone_type_id: int):
+    statement = shape_table.select().where(shape_table.c.shape_type == fire_zone_type_id)
+    result = session.execute(statement).fetchall()
+    return result
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+    fire_zone_shape_type_id = get_fire_zone_unit_shape_type_id(session)
+    zone_units = get_fire_zone_units(session, fire_zone_shape_type_id)
+    tpi_area = calculate_masked_tpi_areas(zone_units)
+    for tuple in tpi_area:
```

Comment:
Minor nit: can we destructure `tuple` into `advisory_shape_id, tpi_enum, fuel_area` to make it a bit easier to read?

Reply:
Good call. Done in https://github.com/bcgov/wps/pull/4241/commits/d5ab94ff6a01f7bb5aca93cc475e0cbd92661e3c